### PR TITLE
Go vet

### PR DESCRIPTION
**Pull request by Fugiman**
`go vet` is a pretty cool tool. I'd highly recommend using it, and requiring code to pass it.

### CODE REVIEW

**Comment by sogko** [comment]
Nice! This is great 👍🏻

I add this to the draft contribution guideline.

/cc @chris-ramon

**Comment by EmergentBehavior** [comment]
Worth running `golint` too while you're at it.

**Comment by Fugiman** [comment]
I'm filing separate branches for each tool. Partially to allow for discussion on each one in particular, and partially because `golint` requires a lot of work to have it start passing :)

File: executor.go [hunk 1]
```diff
@@ -594,10 +594,6 @@ func completeValue(eCtx *ExecutionContext, returnType Type, fieldASTs []*ast.Fie
 	// If field type is Scalar or Enum, serialize to a valid value, returning
 	// null if serialization is not possible.
 	if returnType, ok := returnType.(*Scalar); ok {
-		err := invariant(returnType.Serialize != nil, "Missing serialize method on type")
```
Comment (chris-ramon) [inline executor.go]: I was wondering about this code elimination, but it does totally make sense :+1:  ... we would get compile error if we would make`Scalar.Serialize ` or `Enum.Serialize` actually `nil`.

**Comment by chris-ramon** [comment]
Thanks! @Fugiman :star2: 

Looks good to me.

**Comment by chris-ramon** [comment]
:+

**Pull request merged**